src/ui_basic/panel.cc

```diff
@@ -1078,6 +1105,8 @@ bool Panel::do_key(bool const down, SDL_Keysym const code) {
 		case SDLK_RIGHT:
 		case SDLK_UP:
 		case SDLK_DOWN:
+		case SDLK_F2:
+		case SDLK_F3:
```

Comment:
Is F3 actually used? Either way, it doesn't hurt in here.

Reply:
In debug builds Ctrl+F3 is used to toggle cheating mode on or off